core/storage/user/gae_models_test.py

```diff
@@ -755,21 +755,32 @@ class UserSubscriptionsModelTests(test_utils.GenericTestBase):
     USER_ID_3 = 'user_id_3'
     USER_ID_4 = 'user_id_4'
     CREATOR_IDS = ['4', '8', '16']
+    CREATOR_USERNAMES = ['username4', 'username8', 'username16']
     COLLECTION_IDS = ['23', '42', '4']
     ACTIVITY_IDS = ['8', '16', '23']
     GENERAL_FEEDBACK_THREAD_IDS = ['42', '4', '8']
+    GENERIC_DATETIME = datetime.datetime(2020, 6, 2)
 
     def setUp(self):
         """Set up user models in datastore for use in testing."""
         super(UserSubscriptionsModelTests, self).setUp()
         user_models.UserSubscriptionsModel(id=self.USER_ID_1).put()
 
+        for creator_id in self.CREATOR_IDS:
+            user_models.UserSettingsModel(
+                id=creator_id,
+                gae_id='gae_' + creator_id,
+                username='username' + creator_id,
+                email=creator_id + '@oppia.org'
```

Comment:
Use some random domain instead (e.g. @example.com) to make clear this should be generic

Reply:
Done